code/pizzabot.js

```diff
@@ -1,7 +1,65 @@
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
+const margherita = "Margherita Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+const pizzaVegImg = document.getElementById("veg")
+const pizzaHawImg = document.getElementById("haw")
+const pizzaPepImg = document.getElementById("pep")
+const pizzaMarImg = document.getElementById("mar")
+
+
+console.log(vegetarian, hawaiian, pepperoni, pizzaPrice)
+
+document.getElementById("menu").innerHTML = `Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian}, ${pepperoni}, and ${margherita}.`
+
+
+const calculateTotalCost = (orderQuantity) => {
+  return orderQuantity * pizzaPrice
+}
+
+const calculateCookingTime = (orderQuantity) => {
```

Comment:
That function could be done in a better way. First of all, you don't actually need this local variables `cooking time`. Take a look how we could write same function in a more clean way :
```
const calculateCookingTime = orderQuantity => {
  if (orderQuantity <= 2) {
    return "10 minutes"
  } else if (orderQuantity >= 3 && orderQuantity <= 5) {
    return "15 minutes"
  } else {
    return "20 minutes"
  }
}
```
Regarding clean code, there is also another convention, let me introduce you to it.
Apart from returning given thing from function, `return` keyword does one more thing - it automatically finishes function. That's why we mustn't write anything below `return`, because that code will never be executed (of course in your case we have code below `return`, but it's inside if statement, so it wouldn't enter second statement in the first place).
Thanks to that feature of `return` keyword, we can write our function in slightly different, far more clean fashion :
```
const calculateCookingTime = orderQuantity => {
  if (orderQuantity <= 2) {
    return "10 minutes"
  } else if (orderQuantity >= 3 && orderQuantity <= 5) {
    return "15 minutes"
  }
  return "20 minutes"
}
```
As you can see, we are not writing `else` statement anymore. We can do that thanks to behaviour mentioned above; if result of if statement is lower than 6, we will execute `return "10 minutes"` or `return "15 minutes"` code and finish function. If, on the other hand, result of it was higher or equal 6, we would never enter these two statements and jump straight into line`return "20 minutes"`. After executing that line, we would finish function same way :)